### PR TITLE
Add list / download log file feature to CLI

### PR DESCRIPTION
**Pull request by GDLMadushanka**
Fixes wso2/micro-integrator/issues/1401

### CODE REVIEW

File: cmd/cmd/logFileShow.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    "github.com/wso2/product-mi-tooling/cmd/utils"
+    "github.com/wso2/product-mi-tooling/cmd/utils/artifactUtils"
+    "os"
+)
+
+var logfileName string
+
+// Show Logs command related usage info
+const showLogsCmdLiteral = "show"
+const showLogsCmdShortDesc = "Get information about log files"
```
Comment (arunans23) [inline cmd/cmd/logFileShow.go:33]: Shall we change this to 'Show available log files'?

File: cmd/cmd/logFileShow.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+const showLogsCmdLongDesc = "Download a log file by providing the file name and download location," +
+    " if not provided, list all the log files\n"
+
+var showLogsCmdExamples = "Example:\n" +
+    "To list all the log files\n" +
+    "  " + programName + " " + logsCmdLiteral + " " + showLogsCmdLiteral + "\n\n" +
+    "To download selected log file\n" +
```
Comment (arunans23) [inline cmd/cmd/logFileShow.go:41]: Let's change this to "To download a selected log file"

File: cmd/cmd/logFileShow.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+func printLogsHelp() {
+    fmt.Print(showLogsCmdLongDesc + utils.GetCmdUsage(programName, logsCmdLiteral, showLogsCmdLiteral,
+        "[file-name] --path=[download-location]") + showLogsCmdExamples + utils.GetCmdFlags(logsCmdLiteral))
+}
+
+func executeGetLogsCmd(filename string, targetPath string) {
+    finalUrl, params := utils.GetUrlAndParams(utils.PrefixLogs, "file", filename)
+    utils.UnmarshalLogFileData(finalUrl, nil, params, targetPath+"/"+filename)
```
Comment (arunans23) [inline cmd/cmd/logFileShow.go:91]: "targetPath+"/"+filename", let's add spaces inbetween

File: cmd/cmd/logFileShow.go (hunk 1 above)
Comment (GDLMadushanka) [reply cmd/cmd/logFileShow.go]: +1 fixed

File: cmd/cmd/logFileShow.go (hunk 2 above)
Comment (GDLMadushanka) [reply cmd/cmd/logFileShow.go]: +1 fixed

File: cmd/cmd/logFileShow.go (hunk 3 above)
Comment (GDLMadushanka) [reply cmd/cmd/logFileShow.go]: +1 fixed

**Pull request merged**